OneCommunityGlobal/HighestGoodNetworkApp
# Aaryaneil unit test allTimeEntriesReducer

# Description
Unit test for `src/reducers/allTimeEntriesReducer.js`




## Main changes explained:
Added test cases for the following:
1. should return the initial state when no action is passed
2. should handle GET_ALL_TIME_ENTRIES action and return payload
3. should return the previous state when an unknown action is passed



## How to test:
1. check into current branch
2. do `npm install` and `npm test allTimeEntriesReducer.test.js` to run this PR locally


## Screenshots or videos of changes:
<img width="1291" alt="Screenshot 2024-12-21 at 6 04 19 AM" src="https://github.com/user-attachments/assets/f8dc6b83-8cdd-4ad3-8c8d-163dd3e08035" />
